### DISCUSSION TITLE
¿Consejos para hacer un menú tipo hamburger topnav cuando cambia el tamaño de la pantalla?

### DISCUSSION THREAD

**Discussion by DynamiteFang** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#335] [Q&A]
Hola, amigos,
antes de avanzar más a mi página me gustaría ir metiéndole media queries y modificándola para que sea responsiva a móvil, esto porque la verdad ya cuando está armado el código se me dificulta moverle para que quede al tamaño.
La verdad soy principiante en esto así que quisiera saber si tendrán algún recurso que les haya servido o algún consejo para que el topnav cambie a menú tipo hamburger o a un botón de menú en móvil ya que por alguna razón no me están funcionando los que encuentro.

Intenté con algo de javascript pero al momento de darle click en la página en el VSC me salió ReferenceError: x is not defined at HTMLAnchorElement.onclick así que me encuentro algo perdida en este asunto.

Esto es lo que llevo:
https://dynamitefang.github.io/VacuMex/

**Comment by ChristianDeM** [comment]
Si no te quieres complicar mucho puedes usar  bootsrap  [https://getbootstrap.com/](url)   checa la documentacion de igual forma puedes  bajar ejemplos   [https://getbootstrap.com/docs/5.1/examples/](url)  
te dejo un link de los  nav [https://getbootstrap.com/docs/5.1/components/navs-tabs/#base-nav](url)
